linux_os/guide/system/software/integrity/crypto/configure_bind_crypto_policy/oval/shared.xml

```diff
@@ -6,9 +6,10 @@
         <platform>multi_platform_fedora</platform>
         <platform>Red Hat Enterprise Linux 8</platform>
       </affected>
-      <description>BIND should be configured to use the system-wide crypto policy setting.</description>
+      <description>BIND is not installed or is configured to use the system-wide crypto policy setting.</description>
```

Comment:
This doesn't go here. This goes into OCIL.

Reply:
And really... this change doesn't need to take place in the descriptions.

Reply:
Are you strongly against having the check description like this?

Reply:
Yes.... we don't do this for other checks. And really, this is an OCIL clause statement.

Reply:
Reverted. The check for package bind installed is acting as an applicability check, not as essential part of the check.